content/source/docs/enterprise/api/ptfe_settings.html.md

```diff
@@ -0,0 +1,491 @@
+---
+layout: enterprise2
+page_title: "Private Terraform Settings - API Docs - Terraform Enterprise"
+sidebar_current: "docs-enterprise2-api-private-terraform-settings"
+---
+
+# Private Terraform Settings API
+
+-> **Note**: These API endpoints are in beta and are subject to change. Additionally, these endpoints are only available in Private Terraform instances and only accessible by site administrators.
```

Comment:
If you split this into two separate notes, it'll be easier to scan. (Since the "beta" note appears on every page, it becomes invisible after a few minutes. Moving the "private" note out of its shadow will make it stand out.)